frontend/src/app/projects/project-detail/model-overview/model-overview.component.html

```diff
@@ -4,12 +4,12 @@
  -->
 
 <div class="models" *ngIf="models">
-  <div class="models-title">
-    <h2 id="title">Models</h2>
+  <h2>
+    Models
```

Comment:
The new design on broken on my devices (the old as well). My propsal

HTML:
```
  <div class="models-title">
    <h2 id="title">Models</h2>
    <a mat-button [routerLink]="['/project', project.slug, 'models', 'create']">
      <mat-icon id="add-icon">add</mat-icon>
    </a>
  </div>
```

CSS:
```
.models-title {
  display: flex;
  align-items: center;
}

#title {
  margin: 0;
}
```

and remove the `#title` rule.

Reply:
I do not see that problem, but added the extra div.